spec/services/messages/dead_link_spec.rb

```diff
@@ -7,15 +7,11 @@
     create(
       :flag,
       created_at: Time.zone.local(2021, 8, 1),
-      id: 120,
-      flagger:,
-      project_submission: flagged_submission,
+      project_submission: create(:project_submission, lesson:, user:),
       reason: 'I find it offensive'
     )
   end
 
-  let(:flagger) { create(:user, username: 'OdinUser') }
```

Comment:
**question (non-blocking):** Is `flagger` no longer needed at all? I just didn't understand this one fully 😄

Reply:
Yeah flagger was only used for setting up the flag factory and nowhere else in the test. We can get rid of it with no downsides by relying on the flagger association set up by default when a flagger isn't provided.

Reply:
Gotcha, thanks for clarifying :-)